src/backend/gpopt/translate/CTranslatorDXLToPlStmt.cpp

```diff
@@ -4724,6 +4728,16 @@ CTranslatorDXLToPlStmt::TranslateDXLTblDescrToRangeTblEntry(
 
 	rte->eref = alias;
 
+	/*
+		Store the alias information only when table alias is used to allow 
+		EXPLAIN find it
+	*/
+	if (NULL != md_alias)
```

Comment:
[line](https://github.com/arenadata/gpdb/pull/682/files#diff-2dcfa821b8990b5ae39efff4a97497121dd2e40c31f67d6f180d228f1f34b2c8R4680) dereferences md_alias

Reply:
It does. Initialization of md_alias makes sure that it is not NULL.